.mdl_config.yaml

```diff
@@ -0,0 +1,40 @@
+---
+
+# Default state for all rules
+default": true
```

Comment:
Is the double quote character supposed to be here?

Reply:
Good catch!

Reply:
🦅👀 I've rebased to fix, thanks for the catch!